server/src/main/java/org/elasticsearch/persistent/PersistentTasksNodeService.java

```diff
@@ -112,7 +112,12 @@ public void clusterChanged(ClusterChangedEvent event) {
                         AllocatedPersistentTask persistentTask = runningTasks.get(allocationId);
                         if (persistentTask == null) {
                             // New task - let's start it
-                            startTask(taskInProgress);
+                            try {
+                                startTask(taskInProgress);
+                            } catch (Exception e) {
+                                logger.error("Unable to start allocated task [{}] with id [{}] and allocation id [{}]",
```

Comment:
I think we should at least also log the message. I would opt for stack trace too, but this may not be the convention?

Reply:
Oops, ++ to emitting the exception messages to all of these... not sure why I didn't add it.  Thanks for the catch!